keras_cv/models/__init__.py

```diff
@@ -55,3 +55,5 @@
 from keras_cv.models.resnet_v2 import ResNet101V2
 from keras_cv.models.resnet_v2 import ResNet152V2
 from keras_cv.models.vgg19 import VGG19
+#from keras_cv.models.swin_transformer import SwinTransformer
```

Comment:
@DavidLandup0 It might need to rename `SwinTransformerV1`, [V2](https://arxiv.org/pdf/2111.09883.pdf) is available.